client/components/tinymce/plugins/wpcom-view/views/embed/view.jsx

```diff
@@ -81,7 +73,7 @@ class EmbedView extends Component {
 	}
 
 	setHtml() {
-		if ( ! this.state.body || ! this.refs.iframe ) {
+		if ( ! this.props.embed?.body || ! this.refs.iframe ) {
```

Comment:
It shouldn't be hard to convert the refs from strings to modern `React.createRef`.

Reply:
Sure 👍 I'd prefer doing that in a follow-up PR though, I hope you don't mind? This one is too big anyway, so I try hard to refrain from any non-critical changes 😉

Reply:
Addressing this in https://github.com/Automattic/wp-calypso/pull/39316